src/components/Profile/ProfileImage/ProfileImage.tsx

```diff
@@ -1,20 +1,42 @@
 import "./ProfileImage.css";
-import ProfilePicture from "../../../assets/profile.png";
 import clsx from 'clsx';
+import { useEffect, useState } from "react";
 
 type ImageOptions = {
   circular?: boolean;
+  border?: boolean;
 };
 
+
 const ProfileImage = (options: ImageOptions = {}) => {
+  const [ProfilePicture, setProfilePicture] = useState<string | null>(null);
   const classes = clsx(
     'profile__header__image',
-    { 'profile__header__image__circular': options.circular }
+    { 'profile__header__image__circular': options.circular },
+    { 'profile__header__image__border': options.border }
   );
 
+  useEffect(() => {
+    const loadProfilePicture = async () => {
+      const extensions = ['png', 'jpeg', 'jpg', 'gif', 'webp', 'svg'];
+      for (const extension of extensions) {
+        try {
+          const module = await import(`../../../assets/profile.${extension}`);
+          setProfilePicture(module.default);
+          break;
+        } catch (error) {
+          // Ignore error and try next extension
+        }
+      }
+    };
+
+    loadProfilePicture();
+  }, []);
+
+
   return (
     <div className={classes}>
-      <img src={ProfilePicture} alt="Profile" />
+      <img src={ProfilePicture || ''} alt="Profile" />

```

Comment:
What happens if profile picture is null? 👀 

<details>
<summary>Hint</summary>
<p>
<img width="130" alt="image" src="https://github.com/alcpereira/pdf-generator/assets/48070464/a4fb0c9f-a55d-496f-9c3c-10f1ba5a19bc">
</p>
</details>

Reply:
Should we have a fallback image?